src/main/java/uk/gov/companieshouse/company/links/service/CompanyProfileApiService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.company.links.service;
+
+import org.springframework.beans.factory.annotation.Lookup;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.model.ApiResponse;
+
+//TODO Do we need Api service component for this consumer?
```

Comment:
Yes - need api service component but will have different named methods as not calling insolvency api. Will be calling company-profile-api. e.g. 

```
    /**
     * Invoke Company Profile API.
     */
    public ApiResponse<?> invokeCompanyProfileApi() {
```